flower_classifier/datasets/google/client.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from google_images_search import GoogleImagesSearch
+
+
+def get_secrets():
```

Comment:
This function is generic enough we could reuse it across the two clients if we just pass in `required_keys` as an argument, but we can tackle that in a follow on PR.